docs/reference/contrib/legacy_richtext.rst

```diff
@@ -1,26 +0,0 @@
-.. _legacy_richtext:
-
-=====================
-Legacy richtext
-=====================
-
-.. module:: wagtail.contrib.legacy.richtext
```

Comment:
On its own this didn’t seem to do anything? https://docs.wagtail.io/en/stable/reference/contrib/legacy_richtext.html

Reply:
It think it only converts to a link if the reference (autodoc) has this module. I'm not 100% sure. But can have a look.

In RST/Sphinx there is also a way to cross reference external docs. For example point to Django docs in the same way.